src/main/java/edu/ucsb/cs156/courses/documents/CourseInfo.java

```diff
@@ -18,5 +19,20 @@ public class CourseInfo {
     private String quarter;
     private String courseId;
     private String title;
+    private int contactHours;
     private String description;
-}
\ No newline at end of file
+    private String college;
+    private String objLevelCode;
+    private String subjectArea;
+    private int unitsFixed;
+    private int unitsVariableHigh;
+    private int unitsVariableLow;
+    private boolean delayedSectioning;
+    private boolean inProgressCourse;
+    private String gradingOption;
+    private String instructionType;
+    private boolean onLineCourse;
+    private String deptCode;
+    private List<GeneralEducation> generalEducation;
+    private List<Section> classSections;
+}
```

Comment:
The fields in this document are exactly the same as that of `Course.java`. Why do we have two documents with the exact same member fields?

Reply:
Because of the different formats of functions when calling these variables.